java/ql/integration-tests/all-platforms/kotlin/trap_compression/test.py

```diff
@@ -1,23 +1,28 @@
-from create_database_utils import *
+import runs_on
+import os
+
 
 def check_extensions(directory, counts):
-    if platform.system() == 'Windows':
+    if runs_on.windows:
         # It's important that the path is a Unicode path on Windows, so
         # that the right system calls get used.
-        directory = u'' + directory
+        directory = "" + directory
```

Comment:
while `u"" + directory` did something in python2, this line is unneeded with python 3

Reply:
That change had to be the IDE, I don't remember touching this line at all. I'll remove it and the comment.

Reply:
yeah, in python 2 there was a distinction between `str` and `unicode`, and you had unicode string literals different from normal string literals. In python2 `u"" + x` would convert a string to unicode (although I think `unicode(x)` would have been more idiomatic).

In python 3 `str` is unicode, and `u"bla" is "bla"`, so it stands to reason black and/or your IDE would change that (but keep the unneeded conversion in).